roles/core/log_server/molecule/default/verify.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+  - name: Check that files exists
+    stat:
+      path: "{{ item }}"
+    register: stat_results
+    with_items:
+      - /etc/rsyslog.conf
```

Comment:
Same comment as above for log_client.

Reply:
same comment. Removed and smarter tests.

Reply:
fixed